src/videojs-hls.js

```diff
@@ -279,6 +291,11 @@ videojs.Hls.prototype.play = function() {
     this.mediaIndex = 0;
   }
 
+  if (this.playlists.media() && !this.el().classList.contains('vjs-has-started')) {
```

Comment:
I think classList actually exists in all the browsers we care to support by this project.